src/types.jl

```diff
@@ -104,3 +103,10 @@ macro NestedVector(T::Symbol, N::Int)
     push!(cur_code.args, Symbol(T))
     esc(code)
 end
+
+
+######################
+# Enums for sentiment datasets
+
+@enum setMap train = 1 test = 2
+@enum polarityMap pos = 1 neu = 0 neg = -1
```

Comment:
Can this just be called `Polarity`, and let's spell them out fully:  `positive` etc.